### ISSUE TITLE
Support for Node 20.x (AWS Lambda)

### ISSUE THREAD

**Issue by JustynaStachera** [shelfio/chrome-aws-lambda-layer#62]
Is there any chance to add support for Node 20.x because of the latest AWS update for this Node version?

For Node18.x works like a charm.

**Comment by vladholubiev** [comment]
PRs are welcome! But I guess first we need to wait for support of Node 20.x in the upstream dependency https://github.com/Sparticuz/chromium/

**Comment by tdehollain** [comment]
The release notes of [@sparticuz/chromium v119.0.2](https://www.npmjs.com/package/@sparticuz/chromium) say that it adds support for Node 20, shouldn't it work out of the box?

**Comment by vladholubiev** [comment]
Yes, it works out of the box, as this layer is updated automatically with every release of `sparticuz/chromium`
